ixofoundation/ixo-webclient
# feat: add create claim template module

## Scope
https://nonaredteam.atlassian.net/browse/IXO-839

## Work Done
Add module for creating a claim template
Add initial Redux state 
Add initial UI components

## Steps to Test
Visit `/claim_template/create` and make sure the page loads and you can add questions

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/88378385-2a388f80-cda1-11ea-86aa-53cbbfb996ee.png)
